client/router.js

```diff
@@ -19,7 +19,7 @@ class Router {
       const paramArr = param.split('');
       let _param   = '';
       for (let i = 0; i < paramArr.length; i++) {
-        if (!!~this._specialChars.indexOf(paramArr[i])){
+        if (~this._specialChars.indexOf(paramArr[i])){
```

Comment:
Better to use `.includes()` *Array* method, or keep double negative.

Reply:
sorry, that was my linter probably. I thought i turned that off